src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -0,0 +1,143 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotContainer;
+
+public class DriveSubsystem extends SubsystemBase
+{
+    private DifferentialDrive differentialDrive;
+    private CANPIDController leftPIDController;
+    private CANPIDController rightPIDController;
+    private CANEncoder leftEncoder;
+    private CANEncoder rightEncoder;
+    private double P, I, D, Iz, FF, maxOutput, minOutput;
+    
+    public DriveSubsystem()
+    {
+        CANSparkMax leftMotor1 = new CANSparkMax(0, MotorType.kBrushless);
```

Comment:
Motor objects should be a member of the drive subsystem class then created and assigned in the constructor.